mne/report/report.py

```diff
@@ -1120,6 +1121,15 @@ def add_epochs(
             If ``True``, add PSD plots based on all ``epochs``. If ``False``,
             do not add PSD plots.
         %(projs_report)s
+        image_kwargs : dict | None
+            Keyword arguments to pass to the "epochs image"-generating
+            function (:meth:`mne.Epochs.plot_image`).
+            Needs to be one dict per channel type, for example you could want to use the 
+            rejection limits per channel type:
+            image_kwargs=dict(grad = dict(vmin=-reject['grad'], vmax=-reject['grad']), 
+                                        mag = dict(vmin=reject['mag'], vmax=reject['mag'])))
+
```

Comment:
The double colon is rST magic to say "render as code block" which should look better:
```suggestion
            rejection limits per channel type::

                image_kwargs=dict(
                    grad=dict(vmin=-reject['grad'], vmax=-reject['grad']), 
                    mag=dict(vmin=-reject['mag'], vmax=reject['mag']),
                )

```